account_invoice_refund_link/README.rst

```diff
@@ -90,6 +90,10 @@ Contributors
 
       -  Luis J. Salvatierra <luis.salvatierra@factorlibre.com>
 
+-  `Factor Libre <https://factorlibre.com>`__:
+
+      -  Luis J. Salvatierra <luis.salvatierra@factorlibre.com>
```

Comment:
This contributor is being added at 18.0 migration commit, which doesn't sound correct.